mimikatz/modules/token/revert.md

```diff
@@ -1,6 +1,6 @@
 # revert
 
-`token::revert` reverts to the previous token.
+`token::revert` reverts to the initial token.
```

Comment:
Can you explain (and add to the page) in a few words (or by referring to a link) what you mean by "initial" token?

Reply:
Fixed in fdad34e.

Reply:
It was fixed in https://github.com/ShutdownRepo/The-Hacker-Tools/pull/3/commits/b183db3eabb5cfb57fca24b94b925ceb191e897a